makefile

```diff
@@ -0,0 +1,8 @@
+
+.PHONY: markdownlint markdownlint-fix
+
+markdownlint:
+	docker run -v $(CURDIR):/workdir --rm  ghcr.io/igorshubovych/markdownlint-cli:latest  "**/*.md"
```

Comment:
You might want to pin the version or otherwise the CI might suddenly fail in case of breaking changes which would require you to adapt to them before any other PRs can be merged.

Reply:
thank you, I extracted the image into a variable and pointed it to the most current version (and the only version) :)